aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,45 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor moveaxis(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
+  TORCH_CHECK(src.size() == dst.size(), "Invalid Source or Destination Axes");
```

Comment:
nit: Would be nice to mention what the sizes are and what the operator is so that the user has a better sense of what they're doing.

Something like,
```
TORCH_CHECK(src.size() == dst.size(), "moveaxis: Invalid source or destination dims: src (", src.size(), " dims ) should contain the same number of dims as dst (", dst.size(), "dims)");
```

Reply:
The PyTorch terminology for "axis" is generally "dim", so we should mention "dims" instead of "axis" in the error message.

Reply:
Done. Thanks!